### PR TITLE
CN4260 cel modal button atom

### PR DESCRIPTION
**Pull request by srkela**
PR Checklist:
- [x] I have named the PR correctly
- [x] I have added at least two reviewers
- [x] I have assigned the PR to myself
- [x] I have merged develop into my feature branch and resolved conflicts
- [x] I have tested all new features locally
- [x] I have tested relevant flow locally

### How to name a PR
- If it includes only one issue [CN-1234 Issue name]
- If it includes many issues [CN-1234, CN-2345, CN-3456 What was updated/changed/added]
- If the PR is not ready yet, add [DRAFT] in front of the name

### How to test locally
1. Test happy path
2. Test other paths
3. Remove all warnings

### How to relevant flow locally
- If working on part of CelPay, check the whole happy path

### CODE REVIEW

File: app/components/atoms/CelModalButton/CelModalButton.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  handleBorderRadius = () => {
+    const { position } = this.props;
+
+    switch (position) {
+      case "single":
+        return {
+          borderRadius: 8,
+        };
+      case "left":
+        return {
```
Comment (CrnogoracIvan) [inline app/components/atoms/CelModalButton/CelModalButton.js]: Buttons doesn't have bottom border radius.

File: app/components/atoms/CelModalButton/CelModalButton.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          backgroundColor: STYLES.COLORS.RED,
+          borderColor: STYLES.COLORS.RED,
+        };
+      case "disabled":
+        return {
+          backgroundColor: STYLES.COLORS.DISABLED_BASIC_BUTTON25,
+          borderColor: STYLES.COLORS.DISABLED_BASIC_BUTTON01,
+        };
+      default:
+      case "basic":
```
Comment (CrnogoracIvan) [inline app/components/atoms/CelModalButton/CelModalButton.js]: Remove "case" from default case.

File: app/components/atoms/CelModalButton/CelModalButton.js (hunk 1 above)
Comment (srkela) [reply app/components/atoms/CelModalButton/CelModalButton.js]: Need to have bottom border radius in order to follow modal shape

Comment (CrnogoracIvan) [reply app/components/atoms/CelModalButton/CelModalButton.js]: 👍

**Pull request merged**